app/localization/translated/es.json

```diff
@@ -1510,6 +1510,8 @@
   "NotificationsEnableForm.toggleNotificationsLabel": "Notificaciones automáticas por correo electrónico",
   "NotificationsEnableForm.toggleNotificationsNote": "Enviar notificaciones por correo electrónico al finalizar las ejecuciones",
   "NotificationsEnableForm.typeDescription": "Seleccione una lista de destinatarios {type} para cada regla para enviar notificaciones relacionadas con la ejecución.",
+  "NotificationsEnableForm.unableToConfigure": "Por qué no puedo configurarlo?",
+  "NotificationsEnableForm.unableToConfigureDescription": "Obtenga más información sobre roles y permisos",
```

Comment:
The same for Spanish as we have a Spanish speaking contributor who will help us translate the messages prior to release.